atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -1679,12 +1672,14 @@ private boolean matchingIgnoreCase(TableReference t1, TableReference t2) {
     /**
      * Records the specified metadata for a given table.
      * <p>
-     * Requires all Cassandra nodes to be up and available, otherwise throws an IllegalStateException.
+     * Requires a quorum of Cassandra nodes to be up and available.
      *
      * @param tableRef the name of the table to record metadata for.
      * @param meta a byte array representing the metadata to record.
      *
-     * @throws IllegalStateException if not all hosts respond successfully.
+     * @throws AtlasDbDependencyException if some nodes do not agree on schema version within 1 minute, or there are
+     * fewer than a quorum of reachable nodes.
```

Comment:
javadoc here is confusing: If I have less than quorum nodes up but they all agree that seems to fit both `AtlasDbDependencyException` as well as `InsufficientConsistencyException`